tests/core/tests/test_permissions.py

```diff
@@ -20,6 +22,7 @@ def set_user_book_model_permission(self, action):
         self.user.user_permissions.add(permission)
 
     @override_settings(IMPORT_EXPORT_IMPORT_PERMISSION_CODE="change")
+    @ignore_widget_deprecation_warning
```

Comment:
Thanks for taking the time to review, much appreciated.

Yes, it was a PITA to add it to all those tests.  I tried to set it at the test class level, that would have been a lot easier, but it didn't work because it seems like the decorator has to be set at the test level.  Obviously this is only temporary and I'll remove the decorator when the deprecation ends.

IDK about adding to TestRunner - I think then it would apply to all tests, and at least if applied to only the tests which need it it's a bit more explicit about what's going on.